docs/DeveloperGuide.adoc

```diff
@@ -1199,94 +1186,50 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
 
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |TA |see an overview of events in a week |know what I have that week in a glance
+|`* * *` |new user | be able to use a help command |refer to instructions on what commands are available when
+I forget about them
 
 |`* * *` |TA |set my hourly rate |get the value of my estimated pay according to the latest rate of the semester
 
-|`* * *` |TA |store and retrieve details of my students |I can get details relating to students whenever necessary
+|`* * *` |TA |store contact details of my students |I can contact them with ease whenever
+necessary
 
-|`* * *` |TA |see all my claimable hours in the TSS format |type my claims easily at the end of the semester
+|`* * *` |TA |see all my claimable hours in one place |type my claims easily at the end of the semester
 
 |`* * *` |user |change between the different pages |view the information on the different pages
```

Comment:
help me edit please?

pages should be changed to views for consistency, 
"**switch** between the different **views programmatically**"

Reply:
done